src/components/Person.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
+      {isMarried ? <p className="Person__partner">{`My ${partner}'s name is ${partnerName}`}</p> : 'I am not married'}
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  person: {
```

Comment:
You should add default props for all non required fields in person object
Or make other fields required except the name